build/charts/theia/provisioning/dashboards/flow_records_dashboard.json

```diff
@@ -818,6 +818,168 @@
                 "value": 181
               }
             ]
+          },
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "flowEndReason"
+            },
+            "properties": [
+              {
+                "id": "mappings",
+                "value": [
+                  {
+                    "options": {
+                      "1": {
+                        "index": 0,
+                        "text": "Idle Timeout"
+                      },
+                      "2": {
+                        "index": 1,
+                        "text": "Active Timeout"
+                      },
+                      "3": {
+                        "index": 2,
+                        "text": "End Of Flow"
+                      }
+                    },
+                    "type": "value"
+                  }
+                ]
+              }
+            ]
+          },
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "timeInserted"
+            },
+            "properties": [
+              {
+                "id": "displayName",
+                "value": "time when flow inserted to ClickHouse"
+              }
+            ]
+          },
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "flowStartSeconds"
+            },
+            "properties": [
+              {
+                "id": "displayName",
+                "value": "start time of flow"
+              }
+            ]
+          },
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "flowEndSeconds"
+            },
+            "properties": [
+              {
+                "id": "displayName",
+                "value": "end time of flow"
+              }
+            ]
+          },
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "flowEndSecondsFromSourceNode"
+            },
+            "properties": [
+              {
+                "id": "displayName",
+                "value": "end time of flow reported by source node"
+              }
+            ]
+          },
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "flowEndSecondsFromDestinationNode"
+            },
+            "properties": [
+              {
+                "id": "displayName",
+                "value": "end time of flow reported by destination node"
```

Comment:
"end time of flow reported by destination"

Reply:
Updated